docs/clean.rst

```diff
@@ -395,3 +395,32 @@ use an html5lib filter.
 
 
 .. versionadded:: 2.0
+
+
+Using different CSS parser and sanitizer with `css_style_attr_sanitizer`
+------------------------------------------------------------------------
+
+The argument `css_style_attr_sanitizer` can ``bleach.sanitizer.Cleaner`` be used
+to customize CSS parsing and sanitization. For example, define a new sanitization
+function:
+
+   >>> from bleach import Cleaner
+
+   >>> def sanitize_css(style):
```

Comment:
I'd rename this to `my_sanitize_css` to make it clearer it's the overridden one rather than the Cleaner instance method.